source/faq.txt

```diff
@@ -26,8 +26,8 @@ progress.
 
 For best results, use MongoDB 3.2 or higher, which includes the
 :manual:`$sample </reference/operator/aggregation/sample/>` operator for
-efficient sampling on a collection. On older versions of MongoDB,
-|compass| falls back on a
+efficient :ref:`sampling <sampling>` on a collection. On older versions 
```

Comment:
consider removing the last sentence here if you revise to remove unsupported mdb versions

Reply:
Removed this whole point, actually. Let me know if you think I've gone too far but if 3.6 is the required version the sentence before that one is moot, too.